test/from-draft7/to-2019-09/001-unknown-deprecated.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "title": "deprecated present in the schema",
+        "from": {
+            "deprecated": "newbie"
+        },
+        "to": {
+            "x-deprecated": "newbie"            
```

Comment:
I don't think this is correct. 2019-09 does have the `deprecated` keyword: https://www.learnjsonschema.com/2019-09/meta-data/deprecated/

Reply:
😐 Draft 7 doesn't have `deprecated` keyword, 2019 has. So, if the type od deprecated is object or something invalid then it will be invalid in 2019

Reply:
Ahhhhhhhhh I remember this now. I wonder if we can make the rule names more descriptive, as if I got confused, others might get confused too. Maybe call them i.e. `unknown-before-deprecated` or something like that?

Reply:
I'll say unknown-inp-reviosu-draft-{keyword}..